src/vsr/replica.zig

```diff
@@ -87,7 +88,18 @@ pub fn ReplicaType(
         /// The number of replicas in the cluster:
         replica_count: u8,
 
-        /// The index of this replica's address in the configuration array held by the MessageBus:
+        /// The number of standbys in the cluster.
+        standby_count: u8,
+
+        /// Total amount of nodes (replicas and standbys) in the cluster.
+        ///
+        /// Invariant: node_count = replica_count + standby_count
+        node_count: u8,
```

Comment:
An alternative would be to add `fn node_count` instead of materializing the field, because "single source of truth", but the field seems more readable to me (its presence explains replica_count and standby_count)